### PR TITLE
Resize Images

### LINKED ISSUE #1323: To reach optimal site performance, we should implement properly sized images

### PR DESCRIPTION
**Pull request by actlikewill**
- resizes images

closes #1323

### CODE REVIEW

**Comment by actlikewill** [comment]
@nickmwangemi Perhaps if you have a large screen you can test these images and see if they render okay.

**Comment by longhotsummer** [comment]
Looks good!

File: liiweb/static/images/banner-small.jpg
Comment (longhotsummer) [inline liiweb/static/images/banner-small.jpg:1]: where does this get used?

Comment (longhotsummer) [reply liiweb/static/images/banner-small.jpg:1]: oh never mind, I see -- clever

**Review by longhotsummer — APPROVED** [review]

File: liiweb/static/images/banner-small.jpg
Comment (actlikewill) [reply liiweb/static/images/banner-small.jpg:1]: @longhotsummer The tool that chrome supplies to lint images doesn't like different aspect ratios within a set of `srcset` images. They say that you should not use `srcset` for "art direction". But a small banner image becomes very pixelated, even on moblie, so I only cropped the width. Lighthouse is happy with it.

**Pull request merged**